nutonomy/nuscenes-devkit
# Mtp baseline

* Adds a baseline MTP model with Resnet, MobileNet backbones
* Converts models.py into a directory with physics.py and mtp.py
* Adds unit tests for mtp.py

How to test:
* Run unit tests `nosetests predict/tests` and `nosetests eval/predict/tests`
* The script `run_mtp.py` tests whether the mtp model can overfit on a single example (for 1 or two modes) with the cpu or gpu

@elenacorinagrigore Please verify the model implementation

@emwolff @holger-nutonomy your feedback is appreciated :smile:
